modules/social_features/social_event/modules/social_event_invite/src/SocialEventInviteConfigOverride.php

```diff
@@ -0,0 +1,194 @@
+<?php
+
+namespace Drupal\social_event_invite;
+
+use Drupal\Component\Utility\EmailValidatorInterface;
+use Drupal\Core\Cache\CacheableMetadata;
+use Drupal\Core\Config\ConfigFactoryOverrideInterface;
+use Drupal\Core\Config\StorageInterface;
+use Drupal\Core\Database\Connection;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+/**
+ * Provides an overridden elements.
+ *
+ * @package Drupal\social_event_invite
+ */
+class SocialEventInviteConfigOverride implements ConfigFactoryOverrideInterface {
+
+  /**
+   * The current request.
+   *
+   * @var \Symfony\Component\HttpFoundation\RequestStack
+   */
+  protected $requestStack;
+
+  /**
+   * Email validator.
+   *
+   * @var \Drupal\Component\Utility\EmailValidatorInterface
+   */
+  protected $emailValidator;
+
+  /**
+   * The current active database's master connection.
+   *
+   * @var \Drupal\Core\Database\Connection
+   */
+  protected $database;
+
+  /**
+   * Constructs the configuration override.
+   *
+   * @param \Symfony\Component\HttpFoundation\RequestStack $request_stack
+   *   The request stack.
+   * @param \Drupal\Component\Utility\EmailValidatorInterface $email_validator
+   *   The email validator.
+   * @param \Drupal\Core\Database\Connection $database
+   *   The current active database's master connection.
+   */
+  public function __construct(
+    RequestStack $request_stack,
+    EmailValidatorInterface $email_validator,
+    Connection $database
+  ) {
+    $this->requestStack = $request_stack;
+    $this->emailValidator = $email_validator;
+    $this->database = $database;
+  }
+
+  /**
+   * Load overrides.
+   */
+  public function loadOverrides($names) {
+    $overrides = [];
+
+    $config_name = 'user.settings';
```

Comment:
1. can we skip this part if the user settings already allow users to skip validation?
2. can we create a setting for SM to decide if a GM can actually allow users to skip validation? 
As mentioned in the story, a GM or Event manager can now decide to add anybody and they can skip validation by a SM+ in that case.. I feel like a SM should make a concious decision if they allow this or not.
Especially for those use cases that are much more closed.

Reply:
1) Yes, we can skip it
2) Yes, I think we can add a checkbox  - "Allow user to skip email verification" on the event/group settings pages.

Reply:
done